Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2006-05-16
Initial Package Version: 1.3.13
Upstream Status: Not submitted, CLFS-specific
Origin: copied from the other scripts in ybin directory
Description: Allows ofpath to use PATH_PREFIX like the other scripts.

--- yaboot-1.3.13/ybin/ofpath.orig	2006-05-16 22:32:26.000000000 +0100
+++ yaboot-1.3.13/ybin/ofpath	2006-05-16 22:33:13.000000000 +0100
@@ -26,6 +26,9 @@
 ###############################################################################
 
 PATH="/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin"
+if [ -n "$PATH_PREFIX" ] ; then
+    PATH="${PATH}:${PATH_PREFIX}/sbin:${PATH_PREFIX}/bin:${PATH_PREFIX}/usr/sbin:${PATH_PREFIX}/usr/bin:${PATH_PREFIX}/usr/local/sbin:${PATH_PREFIX}/usr/local/bin"
+fi
 PRG="${0##*/}"
 VERSION=1.0.7
 DEBUG=0
